a2rchi/cli/cli_main.py

```diff
@@ -217,6 +217,7 @@ def cli():
 @click.option('--podman', '-p', 'use_podman', is_flag=True, help="Boolean to use podman instead of docker.")
 @click.option('--gpu', 'use_gpu', is_flag=True, help="Boolean to use GPU for a2rchi. Current support for podman to do this.")
 @click.option('--tag', '-t', 'image_tag', type=str, default=2000, help="Tag for the collection of images you will create to build chat, chroma, and any other specified services")
+@click.option('--hostmode', '-hm', 'host_mode', type=bool, default=False, help="Boolean to use host mode for the containers. This is useful for contianers running on CERN puppet managed machines.")
```

Comment:
Minor comment: For the sake of keeping the code deployment agnostic, I would delete `This is useful for contianers running on CERN puppet managed machines.` or phrase it in a more generic way